rustworkx-core/src/generators/mod.rs

```diff
@@ -26,7 +26,7 @@ mod path_graph;
 mod petersen_graph;
 mod star_graph;
 
-mod utils;
+pub mod utils;
```

Comment:
Doesn't this expose `rustworkx_core::generators::mod::utils` as a public facing interface. Do we want to commit to the interface on the util functions there?

Reply:
This is just there since `rustworkx/src/steiner_tree.rs` was using the `pairwise` fn in `generators.rs`. I moved `pairwise` to utils since it was being used by a couple of the generators, but no longer used in `generators.rs`. If you don't want pub here, the simplest would be to just duplicate `pairwise` in `steiner_tree`.

Reply:
Ah, I missed that when I went through this earlier. I think that's fine but I just wonder about the rest of the utils module. If we only need `pairwise` maybe we should just export only that one function. I'm just worried about committing the API surface for what were previously internal functions in the python side.

Reply:
So where should we put `pairwise` so it's accessible from both `rustworkx` and `rustworkx-core`?

Reply:
I added a top-level utils with pairwise in it. In https://github.com/Qiskit/rustworkx/pull/784/commits/5c3ea62972b3d9adcf62367ae55c14c030a8a270.